autotx/chain_fork.py

```diff
@@ -1,10 +1,11 @@
 import subprocess
-from autotx.utils.ethereum.cached_safe_address import delete_cached_safe_address
+
+from autotx.utils.ethereum import cache
 
 container_name = "autotx_chain_fork"
 
 def start():
-    delete_cached_safe_address()
+    cache.remove("safe.txt")
```

Comment:
NIT: it's better IMO to have a `delete_cached_safe_address()` function rather than have the `"safe.txt"` magic string used throughout the codebase in different files.

Reply:
after re-checking because of your comment, i noticed that we're not using this cached value... we're safe to remove this

this also made me realize that we were not correctly re-using the deployed safe in local environment - left a [comment here](https://github.com/polywrap/AutoTx/pull/69#discussion_r1537831198) mentioning what was happening

Reply:
We were using the cached value, but we're not anymore as of this PR
You removed cache(lambda) which was getting from cache if it exists and calling the lambda and caching if not
Now it calls the func every time and caches without using the cache

Reply:
gotcha - thanks for the clarification @nerfZael i will re-add the logic of caching the safe address